### PR TITLE
Add Hemi style

### PR DESCRIPTION
**Pull request by leanazulyoro**
This Adds the styles as in this figma: https://www.figma.com/file/4fVd9wneclsvYDYD95ApZ9/Hemi-Portal?node-id=738%3A32774&mode=dev

- General look and feel. Creates a new theme and adds the colors required for the design.
- Changes the logo for the full one (Icon + logo)
- Adds the footer with social links
- removes navbar and mobile navbar (the ones that include links to Swap and Pool), we might add them back when we add the Pools features.
- removes the search feature on the top right.

## Screenshots
Captura de pantalla 2024-04-17 a la(s) 17 59 31

Swap:

https://github.com/hemilabs/interface/assets/1864435/3cc0ba70-5bc9-4d26-abba-a4751d5deafb

Send:

https://github.com/hemilabs/interface/assets/1864435/312f952c-bdfa-4547-b1ac-174c90d2b945

### CODE REVIEW

**Review by gabmontes — APPROVED** [review]

File: apps/web/src/components/Footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+import { GithubLogo } from '../Icons/Github'
+import { TwitterXLogo } from '../Icons/TwitterX'
+import { DiscordLogo } from '../Icons/Discord'
+import { LinkedInLogo } from '../Icons/LinkedIn'
+import { HemiLogo } from '../Logo/HemiLogo'
+
+const StyledFooter = styled.div`
```
Comment (gndelia) [inline apps/web/src/components/Footer/index.tsx:8]: should it be `footer` ?
```suggestion
const StyledFooter = styled.footer`
```

File: apps/web/src/components/Footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const Footer = () => {
+  const iconColor = '#AFADAE'
+  const iconSize = 24
+
+  return (
+    <StyledFooter>
+      <div>
+        <HemiLogo width="74" data-testid="uniswap-logo" fill={iconColor} />
+      </div>
```
Comment (gndelia) [inline apps/web/src/components/Footer/index.tsx:31]: should this link to `/` ?

**Review by gndelia — APPROVED** [review]
LGTM, I left some minor suggestions but feel free to merge and ignore or apply in another PR. 

Looks great!

**Comment by leanazulyoro** [comment]
@gndelia since this was approved, I merged it, but I did add the link in the footer in this PR: https://github.com/hemilabs/interface/pull/1

**Pull request merged**